doc/readme.txt

```diff
@@ -31,7 +31,8 @@ reference: generated documentation QBK files (by doxygen_xml2qbk)
 src: examples used in documentation and tools (doxygen_xml2qbk)
 
 Per new algorithm (e.g. foo), one should add:
-1) in file boost/geometry/algorithms/foo.hpp, include a "\ingroup foo" in the doxygen comments
+1) in file boost/geometry/algorithms/foo.hpp, include a "\ingroup foo" in the doxygen comments 
+   (if a directory is created e.g. boost/geometry/algorithms/detail/foo the path should be added in doc/doxy/Doxyfile)
```

Comment:
+1 though this is a different issue than the one mentioned in 1) Shouldn't it be a separate point, e.g. 3) ?

Reply:
OK I added as a separate point, 2) and update the enumeration.